src/message-list/MessageList.js

```diff
@@ -29,10 +29,9 @@ export default class MessageList extends React.PureComponent {
         style={styles.list}
         contentContainerStyle={styles.container}
         automaticallyAdjustContentInset="false"
-        autoScrollToBottom={caughtUp}
+        autoScrollToBottom
```

Comment:
Please leave `caughtUp` in the codebase. I need to fix its behavior but we'll need it